userguide/storage.rst

```diff
@@ -734,10 +734,10 @@ clicking the :guilabel:`Edit Options` button in
 After a dataset is created, click on that dataset and select
 :guilabel:`Create Dataset`, to create a nested dataset, or a
 dataset within a dataset. A zvol can also be created within a dataset.
-When creating datasets, double-check that you are using the
-:guilabel:`Create Dataset` option for the intended volume or dataset.
-If you get confused when creating a dataset on a volume, click all
-existing datasets to close them--the remaining
+When creating datasets, ensure that the :guilabel:`Create Dataset`
+option is being used for the intended volume or dataset.
```

Comment:
These two sentences seem to be saying that the GUI is confusing when creating a dataset.  It probably is, but do what you can to make this explanation clear.

Reply:
I cleaned it up some. However, I'm sure it could be better?